src/lang/ar.js

```diff
@@ -0,0 +1,5 @@
+/* eslint quotes: 0 */
+
+export default {
+	"hello": "مرحبا"
```

Comment:
Should I remove all these `"hello"` lang terms?

Reply:
Hmm, yeah if your initial thing isn't going to have any lang terms maybe we leave adding localization/Serge as an exercise for the next person?